bat/tests/06-test-format-bump/run.bats

```diff
@@ -7,45 +7,137 @@ setup() {
   global_setup
 }
 
-# TODO: This test should run 'mixer build upstream-format' instead
-# once the tooling required to run custom test docker images for
-# is ready.
-@test "Perform upstream format bump on last format boundary" {
+@test "Perform format bump using individual build format-bump <new/old>" {
   upstreamver=$(get-last-format-boundary)
   mixer-init-stripped-down $upstreamver 10
-
-  #Create initial version in the old format
-  mixer-build-all > $LOGDIR/build_all.log
-
-  #Save initial format
   format=$(sed -n 's/[ ]*FORMAT[ ="]*\([0-9]\+\)[ "]*/\1/p' mixer.state)
 
-  #Update to new upstream in the new format
-  mixer-upstream-update $(($upstreamver + 10)) > $LOGDIR/upstream_update.log
+  # create deprecated bundle
+  cat << EOF > local-bundles/editors
+# [TITLE]: editors
+# [DESCRIPTION]: Run popular terminal text editors
+# [STATUS]: Deprecated
+# [CAPABILITIES]:
+# [MAINTAINER]: Test Master <email@example.com>
+EOF
+  mixer bundle add editors
 
-  #Build +20
-  mixer-build-format-bump-new > $LOGDIR/build_formatbump_new.log
+  #Create initial version in the old format
+  mixer-build-bundles > $LOGDIR/build_bundles.log
 
-  #Reset upstream for +10 build
-  echo -n "$upstreamver" > upstreamversion
+  mixer-build-update > $LOGDIR/build_update.log
 
   #Build +10
-  mixer-build-format-bump-old > $LOGDIR/build_formatbump_old.log
+  format=$((format+1))
+  mixer-build-format-bump-old $format > $LOGDIR/build_formatbump_new.log
```

Comment:
The output of the old format bump is stored in build_formatbump_**new**.log

s/new/old  ?